### PR TITLE
Feature/epmldi 1594 elastalert

### PR DESCRIPTION
**Pull request by MaxKavun**
Elastalert for alerting based on elasticsearch metrics
https://jira.epam.com/jira/browse/EPMLDI-1594

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  stage: lint
+  before_script:
+    - git clone https://github.com/lean-delivery/ansible-lint-rules.git ~/ansible-lint-rules
+  script:
+    - yamllint . -c .yamllint
+    - ansible-lint . -c .ansible-lint
+  after_script:
+    - rm -rf ~/ansible-lint-rules
+  tags:
+    - aws
```
Comment (tgadiev) [inline .gitlab-ci.yml]:
```suggestion
    - lint
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  - "3.6"
+
+services:
+  - docker
+
+before_install:
+  - git clone https://github.com/lean-delivery/ansible-lint-rules.git ~/ansible-lint-rules
+
+install:
+  - pip install --upgrade git+https://github.com/ansible/ansible.git@devel docker
```
Comment (tgadiev) [inline .travis.yml]:
```suggestion
  - pip3 install --upgrade ansible==2.8.* docker
```

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - pip install molecule==2.22
```
Comment (tgadiev) [inline .travis.yml]:
```suggestion
  - pip3 install molecule==2.22
```

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  - pip install git+https://github.com/ansible/ansible-lint.git
```
Comment (tgadiev) [inline .travis.yml]:
```suggestion
  - pip3 install git+https://github.com/ansible/ansible-lint.git
```

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+dist: xenial
```
Comment (tgadiev) [inline .travis.yml]:
```suggestion
dist: bionic
```

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+sudo: required
+
+language: python
+python:
+  - "3.6"
```
Comment (tgadiev) [inline .travis.yml]:
```suggestion
  - "3.7"
```

File: molecule/cloud-aws-direct/molecule.yml [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+  options:
+    role-file: requirements.yml
+driver:
+  name: ec2
+lint:
+  name: yamllint
+  options:
+    config-file: .yamllint
+platforms:
+  - name: test-aws-centos7-filebeat
```
Comment (tgadiev) [inline molecule/cloud-aws-direct/molecule.yml]: filebeat?

**Review by tgadiev — CHANGES_REQUESTED** [review]

File: molecule/cloud-aws-direct/molecule.yml (hunk 7 above)
Comment (MaxKavun) [reply molecule/cloud-aws-direct/molecule.yml]: Fixed ! Thanks

File: README.md [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+
+
+### Installing latest elastalert:
+
+
+```yaml
+- name: Install AEM monitoring
+  hosts: all
+  roles:
+    - role: ansible-role-aem-monitoring
```
Comment (tgadiev) [inline README.md]:
```suggestion
    - role: ansible-role-aem-elastalert
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+```
+
+### Installing latest elastalert with alert which has type change and sns notification:
+
+
+```yaml
+- name: Install AEM monitoring
+  hosts: all
+  roles:
+    - role: ansible-role-aem-monitoring
```
Comment (tgadiev) [inline README.md]:
```suggestion
    - role: ansible-role-aem-elastalert
```

File: README.md [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+```
+
+### Installing latest elastalert with alert which has type frequeny and command execution:
+
+
+```yaml
+- name: Install AEM monitoring
+  hosts: all
+  roles:
+    - role: ansible-role-aem-monitoring
```
Comment (tgadiev) [inline README.md]:
```suggestion
    - role: ansible-role-aem-elastalert
```

**Review by tgadiev — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (MaxKavun) [reply README.md]: Fixed, thanks

**Review by tgadiev — APPROVED** [review]

**Pull request merged**